src/app/static/src/app/components/dataExploration/DataExploration.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="content">
+        <div class="pt-4">
+            <adr-integration></adr-integration>
```

Comment:
I'm assuming we should hide ADR Integraton if we're on Step 2, same as for the main app? That's what's shown in the [mockups](https://docs.google.com/presentation/d/1bX7WnIy9Thlub_4Uo17uWlA6f07oicOc/edit#slide=id.p5)